x-pack/platform/packages/shared/kbn-slo-schema/src/rest_specs/routes/find_definition.ts

```diff
@@ -12,6 +12,7 @@ const findSloDefinitionsParamsSchema = t.partial({
   query: t.partial({
     search: t.string,
     includeOutdatedOnly: toBooleanRt,
+    tags: t.union([t.array(t.string), t.string]),
```

Comment:
It should be just a t.string here since we join() on the tags when calling the API.